config.ini.example

```diff
@@ -59,6 +59,11 @@ Whitelist = @example, username, -1001132836449
 ; MediaWhitelist = chatphoto, photo, sticker
 MediaWhitelist = chatphoto, photo, sticker
 
+# dialog type to download. Options are:
+# "user", "group", "channel".
+# An empty list (default if omitted) means all are allowed.
+DialogWhitelist = user, group, channel
```

Comment:
Perhaps `DialogFilter` fits better since we're filtering the type of the dialogs.